log/logging.go

```diff
@@ -291,18 +403,35 @@ func (s *Logger) SetTimestampFlags(flags int) {
 	s.l.SetFlags(flags)
 }
 
+func (s *Logger) SetField(name string, value interface{}) {
```

Comment:
nice!

:nail_care: pls add a comment, especially around the fact this is static.